src/index.js

```diff
@@ -0,0 +1,147 @@
+    Layout.addProject(project);
+  });
+}
+
+Layout.renderView('default');
+
+Layout.newProjectButton.addEventListener('click', () => {
+  Layout.constructor.unselectAllProject();
+  Layout.renderView('newProject');
+});
+Layout.projectForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  event.stopPropagation();
+  const name = Layout.projectFormFieldInput.value;
+  if (name.trim() !== '' && name != null) {
+    const project = new Project(name);
+    const done = APP.addProject(project);
+    if (!done) {
+      // alert('Project already existed');
+      return false;
+    }
+    Layout.projectFormFieldInput.value = '';
+    Layout.addProject(project);
+    APP.storeToLocal();
+    return true;
+  }
+  return false;
+});
+
+Layout.app.addEventListener('submit', (event) => {
+  event.preventDefault();
+  event.stopPropagation();
+  if (event.target.classList.contains('TodoForm')) {
+    const name = Layout.todoFormFieldInput.value;
+    if (name.trim() !== '' && name != null) {
+      const todo = new Todo(name);
+      const added = APP.addTodo(todo);
+      if (!added) {
+        // alert('Todo already existed in this project');
+        return false;
+      }
+      Layout.addTodo(todo);
+      Layout.todoFormFieldInput.value = '';
+      APP.storeToLocal();
+      return true;
+    }
+  }
+
+  if (event.target.classList.contains('TodoEditForm')) {
+    const name = Layout.editTodoFormFieldInput.value;
+    const description = Layout.DescriptionTextArea.value;
+    const date = Layout.DateInput.value;
+    const priority = Layout.PriorityInput.value.toLowerCase();
+    APP.editTodo({
+      name, description, date, priority,
+    });
+    const project = APP.getSelectedProject();
+    Layout.selectProject(project);
+    Layout.renderView('default');
+    Layout.renderView('showProject');
+    APP.storeToLocal();
+  }
+  return false;
+});
+
+Layout.projectList.addEventListener('click', (event) => {
+  if (event.target.classList.contains('project')) {
+    const project = APP.getProject(event.target.id);
+    if (project) {
+      APP.setSelectedProject(project);
+      Layout.selectProject(project);
+      Layout.renderView('default');
+      Layout.renderView('showProject');
+    }
+  }
+  return true;
+});
+
+Layout.app.addEventListener('click', (event) => {
+  if (event.target.classList.contains('todoChecklistLabel')) {
+    const id = event.target.getAttribute('data-id');
+
+    const todo = APP.getTodo(id);
+    if (!todo) {
+      // alert('Todo Not found!');
+      return false;
+    }
+    const todoLi = document.getElementById(id);
+    todoLi.classList.toggle('is-checked');
+    APP.upDateTodoChecked(todo);
+    APP.storeToLocal();
+  }
+
+  if (event.target.classList.contains('todoEditButton') || event.target.classList.contains('fa-pencil')) {
+    const id = event.target.getAttribute('data-id');
+    const todo = APP.getTodo(id);
+    if (!todo) {
+      // alert('Todo Not found!');
```

Comment:
- [ ] Please remove this comment also. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))